### DISCUSSION TITLE
Saw mention of Jellyfin Sync

### DISCUSSION THREAD

**Discussion by NibblesOnGithub** [sbondCo/Watcharr#454] [Q&A] [answered]
But I'm not seeing where/how to enable it. Am I missing a box, or do I need to set up a specific branch?

I just found it in the original issue. I apologize for the distraction

**Comment by IRHM** [comment, marked as answer]
It's no problem @NibblesOnGithub, glad you got it sorted out!

If anyone else comes accross this, currently you need to setup the jellyfin host setting on your server, then login with a jellyfin account for this work (hopefully we can get linking jellyfin accounts to normal accounts working in the future).
